lib/cocoapods/command/lib/lint.rb

```diff
@@ -36,6 +36,7 @@ def self.options
             ['--skip-tests', 'Lint skips building and running tests during validation'],
             ['--test-specs=test-spec1,test-spec2,etc', 'List of test specs to run'],
             ['--analyze', 'Validate with the Xcode Static Analysis tool'],
+            ['--configuration=CONFIGURATION', 'Build using the given configuration (defaults to Release)'],
```

Comment:
can we also add this to `spec lint`? unfortunately the options are duplicated at the moment and have to be added to both